src/Trinity.Core/Network/Messaging/TrinityResponse.cs

```diff
@@ -35,7 +35,7 @@ static TrinityResponse()
 
         internal TrinityResponse(int size)
         {
-            Buffer = (byte*)CMemory.malloc((ulong)size);
+            Buffer = (byte*)CMemory.C_malloc((ulong)size);

```

Comment:
Wondering why we add `C_` prefix on all methods in CMemory, which cause so many changes. I think CMemory.foo is already good for indicating that foo is implemented in Trinity.C.